src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionTypeEnum.java

```diff
@@ -18,4 +21,13 @@ public enum TranscriptionTypeEnum {
     OTHER(999);
 
     private final Integer id;
+
+    public static TranscriptionTypeEnum fromId(Integer id) {
```

Comment:
It was originally in the code before I changed it. It will be needed for the next step in the service which will call it in the same way as the tests.